public/app/features/search/components/SearchItem.test.tsx

```diff
@@ -19,6 +19,7 @@ const data = {
 
 const setup = (propOverrides?: Partial<Props>, renderMethod = shallow) => {
   const props: Props = {
+    //@ts-ignore
```

Comment:
why?

Reply:
It expect `DashboardSearchItemType` as item type vs string, which is probably what I'd have used.